README.md

```diff
@@ -7,9 +7,59 @@
 
 ## Installation
 
-This package is **in development** and will **break often**.  You can install it from the REPL like so:
+This package is **in development** and will **break often**. In order to start using `GeoMakie` make sure to also add a backend (GLMakie, CairoMakie, WGLMakie), namely:
+
+```julia
+] add GLMakie GeoMakie
+```
+
+You may check the installed versions with:
+
+```julia
+] st GLMakie GeoMakie
+```
+
+Start using the package:
+
 ```julia
-]add GeoMakie
+using GLMakie, GeoMakie
 ```
 
-Please see the documentation for examples and basic usage.
\ No newline at end of file
+# Quick start
+
+The following examples are supposed to be self-explanatory. For further information [check out the documentation here!](https://juliaplots.org/GeoMakie.jl/stable/)
+
+### A simple map
+
+```julia
+fig = lines(GeoMakie.coastlines(); color = :black, axis = (; type = GeoAxis))
```

Comment:
We probably shouldn't promote this syntax - it doesn't allow you to customize your projection or limits at all, and as such a user might get pretty confused seeing this first and then a totally different syntax.  My thought is to keep it simple:
```suggestion
fig = Figure()
ga = GeoAxis(f[1, 1])
lines!(ga, GeoMakie.coastlines(); color = :black)
fig
```

That way it's a smooth entry to customizing destination CRS, automatic limit setting, etc.